docs/_sidebar.md

```diff
@@ -1,19 +1,18 @@
-- Teamshares
+- Getting Started
 
-  - [Overview](/teamshares/overview)
+  - [Overview](/)
   - [Contributing](/teamshares/contributing)
   - [Recipe Book](/teamshares/recipes)
   - [Changelog](/teamshares/changelog)
 
-- Getting Started
+- Teamshares Brand
+  - [Logo & Brand Assets](/teamshares/logo-assets)
+  - [Content Guidelines](/teamshares/content)
 
-  - [Overview](/)
-  - [Installation](/getting-started/installation)
```

Comment:
I do think we should leave the `Installation` link, because there are use cases where people might be installing / using this outside of the apps (like on external websites) and would still need this stuff, including the links to our specific libraries